homeassistant/components/tankerkoenig/sensor.py

```diff
@@ -111,9 +113,15 @@ def state(self):
         # key Fuel_type is not available when the fuel station is closed, use "get" instead of "[]" to avoid exceptions
         return self._coordinator.data[self._station_id].get(self._fuel_type)
 
+    @property
+    def unique_id(self) -> str:
+        """Return a unique, friendly identifier for this entity."""
```

Comment:
I wouldn't consider the station_id a "friendly" identifier ;-)
```suggestion
        """Return a unique identifier for this entity."""
```